$OpenBSD$

SPDX-License-Identifier: AGPL-3.0-only

Use ram byte arrays consistently with the vendored simplexmq library.

Index: simplex-chat.cabal
--- simplex-chat.cabal.orig
+++ simplex-chat.cabal
@@ -348,7 +348,7 @@
     , filepath ==1.4.*
     , http-types ==0.12.*
     , http2 >=4.2.2 && <4.3
-    , memory ==0.18.*
+    , ram ==0.22.*
     , mtl >=2.3.1 && <3.0
     , network >=3.1.2.7 && <3.2
     , network-transport ==0.5.6
@@ -539,7 +539,7 @@
     , crypton ==0.34.*
     , directory ==1.3.*
     , filepath ==1.4.*
-    , memory ==0.18.*
+    , ram ==0.22.*
     , mtl >=2.3.1 && <3.0
     , optparse-applicative >=0.15 && <0.17
     , process >=1.6 && <1.6.18
@@ -667,7 +667,7 @@
     , filepath ==1.4.*
     , generic-random ==1.5.*
     , http-types ==0.12.*
-    , memory ==0.18.*
+    , ram ==0.22.*
     , mtl >=2.3.1 && <3.0
     , network ==3.1.*
     , optparse-applicative >=0.15 && <0.17
